Directory.Build.props

```diff
@@ -22,6 +24,11 @@
     <WpfArcadeSdkTargets>$(WpfArcadeSdkPath)Sdk\Sdk.targets</WpfArcadeSdkTargets>
   </PropertyGroup>
 
+  <ItemDefinitionGroup>
+    <!-- We always want MSBuild properties generated that point at the restored location of each package. -->
+    <PackageReference GeneratePathProperty="true" />
```

Comment:
I'm guessing so we can poke at individual binaries? Is our AssetTargetFallback not set correctly?

Reply:
This is so that the build generates the nuget package path variables for us, so I can use them to get the netstandard version of System.IO.Packaging since the NETFX one is a type forwarder only dll to the old WPF assemblies we don't want to load, while the netstandard one has real code for the types that were moved in NET.

Reply:
We shouldn't need that package.  We should just condition all the references and move the code back to WPF.  Depending on what the projects reference that might mean putting back in WindowsBase reference assembly under IFDEF like you've done for source.